phpBB/phpbb/group/helper.php

```diff
@@ -37,4 +78,198 @@ public function get_name($group_name)
 	{
 		return $this->language->is_set('G_' . utf8_strtoupper($group_name)) ? $this->language->lang('G_' . utf8_strtoupper($group_name)) : $group_name;
 	}
+
+	/**
+	 * Get group name details for placing into templates.
+	 *
+	 * @param string	$mode				Can be profile (for getting an url to the profile), group_name (for obtaining the group name), colour (for obtaining the group colour), full (for obtaining a html string representing a coloured link to the group's profile) or no_profile (the same as full but forcing no profile link)
```

Comment:
Please split this explanation onto multiple lines. Ideally your line should be no longer than 120 characters

Reply:
Done